test/test_simulation_results.jl

```diff
@@ -704,3 +730,26 @@ end
         test_simulation_results(file_path, export_path; in_memory = in_memory)
     end
 end
+
+@testset "Test simulation results with system from store" begin
```

Comment:
The changes in this file are minor. I had to shuffle things around so that I could test the new code here.